### DISCUSSION TITLE
How do I restrict the Map to only use Offline Packs?

### DISCUSSION THREAD

**Discussion by jamwise** [maplibre/maplibre-react-native#499] [Q&A]
I have an app that's designed to be offline-first. I've set up my tile server infrastructure with this expectation. But I can't find any way to get the `<MapView />` to **only** use the offline packs. Is there some undocumented way to do this?

**Comment by KiwiKilian** [comment]
I don't yet know an appoach to do this. Did you research if this is possible with MapLibre Native for Android or iOS?
